src/plugins/stake-plugins/generic/util/kilnUtils.ts

```diff
@@ -45,11 +128,55 @@ export const makeKilnApi = (baseUrl: string, apiKey: string): KilnApi => {
 // Cleaners
 // -----------------------------------------------------------------------------
 
+export interface KilnResponse<T> {
+  data: T
+}
 const asKilnResponse = <T>(asT: Cleaner<T>) =>
   asObject({
     data: asT
   })
 
+//
+// Ada
+//
+
+export type AdaStake = ReturnType<typeof asAdaStake>
+const asAdaStake = asObject({
+  wallet_addresses: asArray(asString),
+  stake_address: asString,
+  pool_id: asString,
+  balance: asString,
+  rewards: asString,
+  available_rewards: asString,
+  // delegated_epoch: asNumber,
+  delegated_at: asString,
+  // activated_epoch: asNumber,
+  activated_at: asString,
+  state: asString,
+  net_apy: asNumber,
+  updated_at: asString
+})
+
+export type AdaStakeTransaction = ReturnType<typeof asAdaStakeTransaction>
+const asAdaStakeTransaction = asObject({
+  unsigned_tx_hash: asString,
```

Comment:
unsigned_tx_hash and a handful of keys in asAdaStake aren't used, can they be commented out?